src/util/colors.ts

```diff
@@ -61,6 +62,13 @@ export const theme: IPartialTheme = {
 const colors = {
   textSecondary: deathMetal,
   link: river,
+  river,
+  lake,
+  bay,
+  white,
+  newBlue,
+  indieRock,
+  pond,
```

Comment:
I considered exporting semantically named colors here, but it felt like an unnecessary and potentially confusing extra layer of abstraction.  I found that when I was in `ButtonStyles.ts`, I just wanted to know the actual color names.

Reply:
i think this is fine for now. i wonder if it would make sense for component styles (like ButtonStyles.ts) to pull the contextual color names from `theme.palette` instead of `colors`? or maybe we can just provide additional colors in `theme.palette` and let Fabric components pick up our colors automatically?

Reply:
I gave this some more thought, and I think I now agree that we don't want to spread the yammer color names into the components.  I've updated our colors to be a bit more like fabric's, with both a `palette` that has names like `themePrimary`, `neutralDark` etc. AND `semanticColors` that have `bodySubtext`, `link` etc.  Both of these, using the types defined by fabric.

For component level styles, I think we should try to use a semanticColors if the name fits well, and if not, then use a color from the palette.